src/processor.py

```diff
@@ -124,7 +124,7 @@ def __call__(self, list_of_str: List[str]) -> Dict[str, List[int]]:
                     truncation=False,
                     return_attention_mask=False,
                     return_token_type_ids=False,
-                    add_special_tokens=True,
+                    add_special_tokens=False,
```

Comment:
In `GPT2TokenizerFast`, `build_inputs_with_special_tokens` method in build`prepare_for_model` is no overrided. therefore, `add_special_tokens=True` and `add_special_tokens=False` are not different. 
to: @bzantium 

- `GPT2TokenizerFast`
https://github.com/huggingface/transformers/blob/95119ad7b0369d76925cea12ffb29d3c31014570/src/transformers/models/gpt2/tokenization_gpt2_fast.py#L70-L187

- `PreTrainedTokenizerBase`
https://github.com/huggingface/transformers/blob/95119ad7b0369d76925cea12ffb29d3c31014570/src/transformers/tokenization_utils_base.py#L2826-L2844

Reply:
But I think that `add_special_tokens=False` is better than the previous thing in semantic context. I resolve this conversation. Thanks @bzantium